HW1/task2.py

```diff
@@ -0,0 +1,9 @@
+#Задача 2 (4): Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+#Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+#а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+print("Введите сколько нужно сделать журавликов ребятам: ")
+S = int(input())
+a = int ((S / 3)/2)
```

Comment:
```suggestion
a = int (S / 3 / 2)
```
Но функцию `int` вообще использовать не обязательно, можно использовать целочисленное деление.